lib/api/controllers/routerController.js

```diff
@@ -153,13 +153,13 @@ module.exports = function RouterController (kuzzle) {
         {verb: 'delete', url: '/:index/:collection/:id', controller: 'write', action: 'delete'},
         {verb: 'delete', url: '/:index/:collection', controller: 'admin', action: 'deleteCollection'},
 
-        {verb: 'put', url: '/roles/:id', controller: 'security', action: 'putRole'},
-        {verb: 'put', url: '/profiles/:id', controller: 'security', action: 'putProfile'},
+        {verb: 'put', url: '/roles/:id', controller: 'security', action: 'createOrReplaceRole'},
+        {verb: 'put', url: '/profiles/:id', controller: 'security', action: 'createOrReplaceProfile'},
         {verb: 'put', url: '/:index', controller: 'admin', action: 'createIndex'},
         {verb: 'put', url: '/:index/:collection', controller: 'write', action: 'createCollection'},
-        {verb: 'put', url: '/:index/:collection/_mapping', controller: 'admin', action: 'putMapping'},
+        {verb: 'put', url: '/:index/:collection/_mapping', controller: 'admin', action: 'createOrReplaceMapping'},
```

Comment:
This route cannot be named `createOrReplace`, since we cannot replace a whole mapping.
This route behave exactly like an `update` does, so I suggest we name it like that.